stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-7.14
+resolver: lts-9.20
```

Comment:
We leave this at the lowest LTS bound supporting `7.6` and rely on testing to check for incompatibility with the later LTS resolvers. https://github.com/sdiehl/protolude/blob/master/all_stack.sh